.github/workflows/e2e-injection-test.yml

```diff
@@ -0,0 +1,59 @@
+name: E2E Injection Test
+
+
+on:
+  workflow_dispatch:
```

Comment:
Just looking through the docs, I think we can remove the `:` here https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch